### PR TITLE
Add DNS for the BOD VPC

### PR DESCRIPTION
**Pull request by jsf9k**
* Add DNS for the BOD VPC
* Change the names of some CyHy Terraform variables so they are distinct from the corresponding BOD variables.

Note that I do not use fixed IPs like the CyHy VPC does, but this still appears to work.

### CODE REVIEW

File: terraform/bod_private_dns.tf [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  ]
+}
+
+resource "aws_route53_record" "bod_bastion_A" {
+  zone_id = "${aws_route53_zone.bod_private_zone.zone_id}"
+  name = "bastion.${aws_route53_zone.bod_private_zone.name}"
+  type = "A"
+  ttl = 300
+  records = [
+    "${aws_instance.bod_bastion.public_ip}"
```
Comment (jsf9k) [inline terraform/bod_private_dns.tf]: Should this be the private IP?  I copied what was done in the CyHy VPC, but this doesn't seem correct.

Comment (dav3r) [reply terraform/bod_private_dns.tf]: @felddy, care to comment about this?

Comment (felddy) [reply terraform/bod_private_dns.tf]: I'm trying to find where @jsf9k  copied this from.  The private bastion entries I wrote don't reference the `public_ip`:
https://github.com/dhs-ncats/cyhy_amis/blob/bb304156750f0028b453ff6bef94bca92b9da700/terraform/cyhy_private_dns.tf#L35

Also, the entries in route53 look correct in production.

Comment (jsf9k) [reply terraform/bod_private_dns.tf]: You are correct sir. I don't know where I got that from.

**Comment by jsf9k** [comment]
@felddy, can you also comment on the fixed IP issue?  That didn't seem to be an issue for me.

**Review by dav3r — APPROVED** [review]
These changes all make sense and look good to me.

**Pull request merged**